totpauth-impl/src/main/resources/views/totp-register.vm

```diff
@@ -58,6 +59,13 @@ p.error {
             </div>
           </form>
           <li><a href="https://support.google.com/accounts/answer/1066447" target="_blank">Google Authenticator installation instructions</a></li>
+<li><a href="https://support.authy.com/hc/en-us/articles/115001945848-Installing-Authy-apps" target="_blank">Authy Authenticator installation instructions</a></li>
```

Comment:
Would this be a better link?

https://support.authy.com/hc/en-us/articles/360006303934-Add-a-New-Two-Factor-Authentication-2FA-Account-Token-in-the-Authy-App

The current link just talks about how to install Authy

Reply:
I agree

Reply:
Change made